lib/Button/Button.less

```diff
@@ -16,6 +27,14 @@
 	padding: 0 @moon-button-h-padding;
 	margin: 0 @moon-spotlight-outset;
 	color: @moon-button-text-color;
+	-webkit-transform: translateZ(0);
+	-webkit-transition: transform 0.2s ease-out;
+	transform: translateZ(0);
```

Comment:
Instead of explicitly adding the `translateZ(0)` here, you can use the `.moon-composite` class I added during the lite work. It has that as well as `will-change` so we can consolidate the layer promotion into a single place.